main.py

```diff
@@ -88,15 +90,18 @@ def get_private_channel(user):
             return pc
     return None
 
-def print_group_info(channel, groupInfo):
+def print_group_info(channel, group):

```

Comment:
You changed the argument but not all the references to the variable in the function. Also the reason the argument was not called 'group' is because then is overloads the variable group because it is also the name of the class that is imported